tests/builder/test_base.py

```diff
@@ -1,9 +1,25 @@
 import sys
+from pathlib import Path
 from unittest import mock
 
+import pytest
+
 from rpmlb.builder.base import BaseBuilder
 
 
+@pytest.fixture
+def empty_spec_path(tmpdir):
+    """Empty spec file path"""
```

Comment:
`tmpdir` is a built-in fixture provided by the `pytest` framework itself. I did not see the `push_tmp_dir` function, since I did not look for it and just used the built-in one.

`empty_spec_path` ensures freshly created empty spec file for each test case that would need it. I agree that the pattern is similar to `push_tmp_dir`, but I'm not aware on how to implement that generically enough to be usable for both files and dictionaries at the same time.